ppb/camera.py

```diff
@@ -163,7 +163,11 @@ def translate_point_to_screen(self, point: Vector) -> Vector:
         :return: A vector in pixels.
         :rtype: Vector
         """
-        return Vector(point.x - self.left, self.top - point.y) * self.pixel_ratio
+
+        try:
+          return Vector(point.x - self.left, self.top - point.y) * self.pixel_ratio
+        except AttributeError as error:
+          raise TypeError("Expected Vector")
```

Comment:
```suggestion
          raise TypeError(f"Expected Vector got {type(point).__name__}: {point}")
```

Got some feedback from @AstraLuma, this should cover it.

Reply:
`{point!r}` to get the repr string